geo-types/src/polygon.rs

```diff
@@ -387,6 +387,7 @@ where
     // The polygon is convex if the z-components of the cross products are either
     // all positive or all negative. Otherwise, the polygon is non-convex.
     // see: http://stackoverflow.com/a/1881201/416626
+    #[deprecated(since = "0.6.1", note = "Please use `geo::is_convex` instead")]
```

Comment:
Seems like currently there is no `impl IsConvex for Polygon` in `geo::algorithm::IsConvex`, so we can't call `geo::is_convex(&poly)`.

Is there a reason not to add that impl?

And if there is some reason to omit that Impl, then is the suggestion here, more explicitly, to do something like `self.exterior.is_convex() && self.interios.is_empty()`?

Reply:
Great catch!  The mathematical definition of convex for polygons is a bit different from what is provided as `is_convex` in geo-types.  In particular, the inner holes are not considered.  I've updated the doc to suggest is_convex on the exterior.